src/providers/aws.rs

```diff
@@ -135,15 +135,19 @@ impl MachineSetup {
 /// Each individual region is handled by `AWSRegion`.
 pub struct AWSLauncher<P: ProvideAwsCredentials> {
     credential_provider: Option<Box<dyn Fn() -> Result<P, Error>>>,
-    max_instance_duration: Option<std::time::Duration>,
+    max_instance_duration: std::time::Duration,
```

Comment:
Maybe we should just store this as a number of hours, since it can't be set more granuarly than that anyway?